ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: 'maven-publish'
  */
 ext {
     // String property to store version name.
-    stringVersion = "6.7.0.0"
+    stringVersion = "6.4.13.0"
```

Comment:
not sure what version it will be

Reply:
I saw this PR depend on native SDK `vungle-android-sdk-6.7.0-dax`, should `stringVersion` change to 6.7.0.1?

Reply:
its more close to 6.7.0 based release than 6.4.0 , am not sure if this version allows any alpha numeric like "6.7.0.0-NAX" (No AndroidX)for

Reply:
No, the version should all in the Integer type.
https://github.com/Vungle/googleads-mobile-android-mediation/blob/3b8db930f3903790b8cd12a729a0c6cb47022579/ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java#L58-L60

Reply:
This will be passed to publishers who used 6.4.11 and depends on old AdMob.
But I'm ok with 6.7.0.1 or even 6.7.1.1(if we could pick all the 6.7.1 fixes from Vungle SDK)